src/preferences/dialog/dlgpreferences.cpp

```diff
@@ -61,6 +65,50 @@ DlgPreferences::DlgPreferences(
           m_pConfig(pSettingsManager->settings()),
           m_pageSizeHint(QSize(0, 0)) {
     setupUi(this);
+
```

Comment:
I'd like to have this extracted into a helper function. When reading the `DlgPreferences` constructot I just need to know that we `fixSlidersAppareance();`. I don't need to know the details.

Reply:
Gràcies, Ferran, good point. I have moved the code to a private method as suggested.